trillian/integration/ct_integration.go

```diff
@@ -682,6 +684,9 @@ func makePreIssuerPrecertChain(chain []ct.ASN1Cert, issuer *x509.Certificate, si
 	preIssuer.PublicKeyAlgorithm = x509.ECDSA
 	preIssuer.PublicKey = preSigner.Public()
 	preIssuer.ExtKeyUsage = append(preIssuer.ExtKeyUsage, x509.ExtKeyUsageCertificateTransparency)
+	// Set a different subject-key-id for the intermediate.  Note that this needs to be the DER-encoded
+	// SEQUENCE, not just the key ID.
+	preIssuer.SubjectKeyId = []byte{0xde, 0xad, 0xbe, 0xaf}
```

Comment:
They just need to be different than the subject ID of the real issuer (which is 0x01020304)